env_support/cmake/customlib.cmake

```diff
@@ -0,0 +1,183 @@
+# Option to build shared libraries (as opposed to static), default: OFF
+option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
+
+# Build LVGL library
+add_library(lvgl ${_LVGL_SOURCES})
```

Comment:
What is the exact purpose of this file? Is it for 3rd party libs? If so we should confider the proposal from here too https://github.com/lvgl/lvgl/pull/6265/files